window_installer/__manifest__.py

```diff
@@ -0,0 +1,63 @@
+{
+    'name': 'Window Installation',
+    'version': '17.4',
```

Comment:
```suggestion
    'version': '1.0,
```
This is not related to the version of Odoo